<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="extrudeButton">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>Extrude</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MyGLWidget" name="myGLWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>480</width>
         <height>480</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="animateButton">
     <property name="text">
      <string>Animate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignTop">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <widget class="QLabel" name="eyeLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="camLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="upLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>setYRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
